src/winstanley/references/WdlTaskDeclarationReference.scala

```diff
@@ -0,0 +1,42 @@
+package winstanley.references
+
+import javax.annotation.Nullable
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.{PsiElement, PsiReferenceBase}
+import winstanley.psi.WdlCallableLookup
+import winstanley.structure.WdlImplicits._
+
+final case class WdlTaskDeclarationReference(value: WdlCallableLookup) extends PsiReferenceBase[PsiElement](value, value.getTextRange) {
```

Comment:
To match the `value`, it's probably best to call this `WdlCallableDeclarationReference `